### PR TITLE
Anko - Change Menu Screen To Anko

### PR DESCRIPTION
**Pull request by at-toannguyen1**
#### Related issues (Required)
- Task #113 
- Bug #

#### Description (Required)
- [x] Change custom menu from XML to AnkoLayout

### CODE REVIEW

File: app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontal.kt [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+ * Custom floating button menu
+ * Created by haingoq on 29/09/2017.
+ */
+class FloatingButtonHorizontal(
+        context: Context, attrs: AttributeSet? = null) :
+        LinearLayout(context, attrs), View.OnClickListener {
+    private lateinit var ui: FloatingButtonHorizontalUI
+
+    init {
+        ui= FloatingButtonHorizontalUI()
```
Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontal.kt]: WARNING:

File: app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+    internal lateinit var imgBtnMenu: ImageButton
+    override fun createView(ui: AnkoContext<ViewGroup>): View = with(ui) {
+        verticalLayout {
+            lparams(wrapContent, wrapContent)
+            relativeLayout {
+                imageButton {
+                    id = ID_BTN_SEARCH
+                    imageResource = R.drawable.ic_search
+                    backgroundResource = R.drawable.custom_bg_item_search_button
+                }.lparams(dip(50), dip(50)) {
```
Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt]: WARNING: Report magic numbers. Magic number is a numeric literal that is not defined as a constant and hence it's unclear what the purpose of this number is. It's better to declare such numbers as constants and give them a proper name. By default, -1, 0, 1, and 2 are not considered to be magic numbers.

Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt]: WARNING: Report magic numbers. Magic number is a numeric literal that is not defined as a constant and hence it's unclear what the purpose of this number is. It's better to declare such numbers as constants and give them a proper name. By default, -1, 0, 1, and 2 are not considered to be magic numbers.

File: app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+                    alignParentRight()
+                    rightMargin = dip(5)
```
Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt]: WARNING: Report magic numbers. Magic number is a numeric literal that is not defined as a constant and hence it's unclear what the purpose of this number is. It's better to declare such numbers as constants and give them a proper name. By default, -1, 0, 1, and 2 are not considered to be magic numbers.

File: app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+                }
+                textView(R.string.custom_floating_menu_search_title) {
+                    leftPadding = dip(10)
```
Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt]: WARNING: Report magic numbers. Magic number is a numeric literal that is not defined as a constant and hence it's unclear what the purpose of this number is. It's better to declare such numbers as constants and give them a proper name. By default, -1, 0, 1, and 2 are not considered to be magic numbers.

File: app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+                    rightPadding = dip(10)
```
Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt]: WARNING: Report magic numbers. Magic number is a numeric literal that is not defined as a constant and hence it's unclear what the purpose of this number is. It's better to declare such numbers as constants and give them a proper name. By default, -1, 0, 1, and 2 are not considered to be magic numbers.

File: app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt [hunk 6]
```diff
@@ -0,0 +1,172 @@
...
+                    textSize = px2dip(dimen(R.dimen.custom_menu_text_size))
+                    backgroundResource = R.drawable.custom_bg_item_menu_title
+                }.lparams(wrapContent, dip(30)) {
```
Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt]: WARNING: Report magic numbers. Magic number is a numeric literal that is not defined as a constant and hence it's unclear what the purpose of this number is. It's better to declare such numbers as constants and give them a proper name. By default, -1, 0, 1, and 2 are not considered to be magic numbers.

File: app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt [hunk 7]
```diff
@@ -0,0 +1,172 @@
...
+                    centerVertically()
+                    rightMargin = 10
```
Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt]: WARNING: Report magic numbers. Magic number is a numeric literal that is not defined as a constant and hence it's unclear what the purpose of this number is. It's better to declare such numbers as constants and give them a proper name. By default, -1, 0, 1, and 2 are not considered to be magic numbers.

File: app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt [hunk 8]
```diff
@@ -0,0 +1,172 @@
...
+                    leftOf(ID_BTN_SEARCH)
+                    gravity = Gravity.CENTER
+                }
+            }.lparams(dip(150), wrapContent) {
```
Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt]: WARNING: Report magic numbers. Magic number is a numeric literal that is not defined as a constant and hence it's unclear what the purpose of this number is. It's better to declare such numbers as constants and give them a proper name. By default, -1, 0, 1, and 2 are not considered to be magic numbers.

File: app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt [hunk 9]
```diff
@@ -0,0 +1,172 @@
...
+                visibility = View.VISIBLE
+            }
+            relativeLayout {
+                imageButton {
+                    id = ID_BTN_GROUP
+                    imageResource = R.drawable.ic_group_white_24dp
+                    backgroundResource = R.drawable.custom_bg_item_search_button
+                }.lparams(dip(50), dip(50)) {
```
Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt]: WARNING: Report magic numbers. Magic number is a numeric literal that is not defined as a constant and hence it's unclear what the purpose of this number is. It's better to declare such numbers as constants and give them a proper name. By default, -1, 0, 1, and 2 are not considered to be magic numbers.

Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt]: WARNING: Report magic numbers. Magic number is a numeric literal that is not defined as a constant and hence it's unclear what the purpose of this number is. It's better to declare such numbers as constants and give them a proper name. By default, -1, 0, 1, and 2 are not considered to be magic numbers.

File: app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt [hunk 10]
```diff
@@ -0,0 +1,172 @@
...
+                    alignParentRight()
+                    rightMargin = dip(5)
```
Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt]: WARNING: Report magic numbers. Magic number is a numeric literal that is not defined as a constant and hence it's unclear what the purpose of this number is. It's better to declare such numbers as constants and give them a proper name. By default, -1, 0, 1, and 2 are not considered to be magic numbers.

File: app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt [hunk 11]
```diff
@@ -0,0 +1,172 @@
...
+                }
+                textView(R.string.custom_floating_menu_group_title) {
+                    leftPadding = dip(10)
```
Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt]: WARNING: Report magic numbers. Magic number is a numeric literal that is not defined as a constant and hence it's unclear what the purpose of this number is. It's better to declare such numbers as constants and give them a proper name. By default, -1, 0, 1, and 2 are not considered to be magic numbers.

File: app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt [hunk 12]
```diff
@@ -0,0 +1,172 @@
...
+                    rightPadding = dip(10)
```
Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt]: WARNING: Report magic numbers. Magic number is a numeric literal that is not defined as a constant and hence it's unclear what the purpose of this number is. It's better to declare such numbers as constants and give them a proper name. By default, -1, 0, 1, and 2 are not considered to be magic numbers.

File: app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt [hunk 13]
```diff
@@ -0,0 +1,172 @@
...
+                    textSize = px2dip(dimen(R.dimen.custom_menu_text_size))
+                    backgroundResource = R.drawable.custom_bg_item_menu_title
+                }.lparams(wrapContent, dip(30)) {
```
Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt]: WARNING: Report magic numbers. Magic number is a numeric literal that is not defined as a constant and hence it's unclear what the purpose of this number is. It's better to declare such numbers as constants and give them a proper name. By default, -1, 0, 1, and 2 are not considered to be magic numbers.

File: app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt [hunk 14]
```diff
@@ -0,0 +1,172 @@
...
+                    centerVertically()
+                    rightMargin = 10
```
Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt]: WARNING: Report magic numbers. Magic number is a numeric literal that is not defined as a constant and hence it's unclear what the purpose of this number is. It's better to declare such numbers as constants and give them a proper name. By default, -1, 0, 1, and 2 are not considered to be magic numbers.

File: app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt [hunk 15]
```diff
@@ -0,0 +1,172 @@
...
+                    leftOf(ID_BTN_GROUP)
+                    gravity = Gravity.CENTER
+                }
+            }.lparams(dip(150), wrapContent) {
```
Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt]: WARNING: Report magic numbers. Magic number is a numeric literal that is not defined as a constant and hence it's unclear what the purpose of this number is. It's better to declare such numbers as constants and give them a proper name. By default, -1, 0, 1, and 2 are not considered to be magic numbers.

File: app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt [hunk 16]
```diff
@@ -0,0 +1,172 @@
...
+                visibility = View.VISIBLE
+            }
+            relativeLayout {
+                imageButton {
+                    id = ID_BTN_CALENDAR
+                    backgroundResource = R.drawable.custom_bg_item_calendar_button
+                    imageResource = R.drawable.ic_calendar
+                }.lparams(dip(50), dip(50)) {
```
Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt]: WARNING: Report magic numbers. Magic number is a numeric literal that is not defined as a constant and hence it's unclear what the purpose of this number is. It's better to declare such numbers as constants and give them a proper name. By default, -1, 0, 1, and 2 are not considered to be magic numbers.

Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt]: WARNING: Report magic numbers. Magic number is a numeric literal that is not defined as a constant and hence it's unclear what the purpose of this number is. It's better to declare such numbers as constants and give them a proper name. By default, -1, 0, 1, and 2 are not considered to be magic numbers.

File: app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt [hunk 17]
```diff
@@ -0,0 +1,172 @@
...
+                    alignParentRight()
+                    rightMargin = dip(5)
```
Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt]: WARNING: Report magic numbers. Magic number is a numeric literal that is not defined as a constant and hence it's unclear what the purpose of this number is. It's better to declare such numbers as constants and give them a proper name. By default, -1, 0, 1, and 2 are not considered to be magic numbers.

File: app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt [hunk 18]
```diff
@@ -0,0 +1,172 @@
...
+                }
+                textView(R.string.custom_floating_menu_calendar_title) {
+                    leftPadding = dip(10)
```
Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt]: WARNING: Report magic numbers. Magic number is a numeric literal that is not defined as a constant and hence it's unclear what the purpose of this number is. It's better to declare such numbers as constants and give them a proper name. By default, -1, 0, 1, and 2 are not considered to be magic numbers.

File: app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt [hunk 19]
```diff
@@ -0,0 +1,172 @@
...
+                    rightPadding = dip(10)
```
Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt]: WARNING: Report magic numbers. Magic number is a numeric literal that is not defined as a constant and hence it's unclear what the purpose of this number is. It's better to declare such numbers as constants and give them a proper name. By default, -1, 0, 1, and 2 are not considered to be magic numbers.

File: app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt [hunk 20]
```diff
@@ -0,0 +1,172 @@
...
+                    textSize = px2dip(dimen(R.dimen.custom_menu_text_size))
+                    backgroundResource = R.drawable.custom_bg_item_menu_title
+                }.lparams(wrapContent, dip(30)) {
```
Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt]: WARNING: Report magic numbers. Magic number is a numeric literal that is not defined as a constant and hence it's unclear what the purpose of this number is. It's better to declare such numbers as constants and give them a proper name. By default, -1, 0, 1, and 2 are not considered to be magic numbers.

File: app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt [hunk 21]
```diff
@@ -0,0 +1,172 @@
...
+                    centerVertically()
+                    rightMargin = 10
```
Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt]: WARNING: Report magic numbers. Magic number is a numeric literal that is not defined as a constant and hence it's unclear what the purpose of this number is. It's better to declare such numbers as constants and give them a proper name. By default, -1, 0, 1, and 2 are not considered to be magic numbers.

File: app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt [hunk 22]
```diff
@@ -0,0 +1,172 @@
...
+                    leftOf(ID_BTN_CALENDAR)
+                    gravity = Gravity.CENTER
+                }
+            }.lparams(dip(150), wrapContent) {
```
Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt]: WARNING: Report magic numbers. Magic number is a numeric literal that is not defined as a constant and hence it's unclear what the purpose of this number is. It's better to declare such numbers as constants and give them a proper name. By default, -1, 0, 1, and 2 are not considered to be magic numbers.

File: app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt [hunk 23]
```diff
@@ -0,0 +1,172 @@
...
+                visibility = View.VISIBLE
+            }
+            relativeLayout {
+                imageButton {
+                    id = ID_BTN_PROFILE
+                    imageResource = R.drawable.ic_profile
+                    backgroundResource = R.drawable.custom_bg_item_profile_button
+                }.lparams(dip(50), dip(50)) {
```
Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt]: WARNING: Report magic numbers. Magic number is a numeric literal that is not defined as a constant and hence it's unclear what the purpose of this number is. It's better to declare such numbers as constants and give them a proper name. By default, -1, 0, 1, and 2 are not considered to be magic numbers.

Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt]: WARNING: Report magic numbers. Magic number is a numeric literal that is not defined as a constant and hence it's unclear what the purpose of this number is. It's better to declare such numbers as constants and give them a proper name. By default, -1, 0, 1, and 2 are not considered to be magic numbers.

File: app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt [hunk 24]
```diff
@@ -0,0 +1,172 @@
...
+                    alignParentRight()
+                    rightMargin = dip(5)
```
Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt]: WARNING: Report magic numbers. Magic number is a numeric literal that is not defined as a constant and hence it's unclear what the purpose of this number is. It's better to declare such numbers as constants and give them a proper name. By default, -1, 0, 1, and 2 are not considered to be magic numbers.

File: app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt [hunk 25]
```diff
@@ -0,0 +1,172 @@
...
+                }
+                textView(R.string.custom_floating_menu_search_title) {
+                    leftPadding = dip(10)
```
Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt]: WARNING: Report magic numbers. Magic number is a numeric literal that is not defined as a constant and hence it's unclear what the purpose of this number is. It's better to declare such numbers as constants and give them a proper name. By default, -1, 0, 1, and 2 are not considered to be magic numbers.

File: app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt [hunk 26]
```diff
@@ -0,0 +1,172 @@
...
+                    rightPadding = dip(10)
```
Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt]: WARNING: Report magic numbers. Magic number is a numeric literal that is not defined as a constant and hence it's unclear what the purpose of this number is. It's better to declare such numbers as constants and give them a proper name. By default, -1, 0, 1, and 2 are not considered to be magic numbers.

File: app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt [hunk 27]
```diff
@@ -0,0 +1,172 @@
...
+                    textSize = px2dip(dimen(R.dimen.custom_menu_text_size))
+                    backgroundResource = R.drawable.custom_bg_item_menu_title
+                }.lparams(wrapContent, dip(30)) {
```
Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt]: WARNING: Report magic numbers. Magic number is a numeric literal that is not defined as a constant and hence it's unclear what the purpose of this number is. It's better to declare such numbers as constants and give them a proper name. By default, -1, 0, 1, and 2 are not considered to be magic numbers.

File: app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt [hunk 28]
```diff
@@ -0,0 +1,172 @@
...
+                    centerVertically()
+                    rightMargin = 10
```
Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt]: WARNING: Report magic numbers. Magic number is a numeric literal that is not defined as a constant and hence it's unclear what the purpose of this number is. It's better to declare such numbers as constants and give them a proper name. By default, -1, 0, 1, and 2 are not considered to be magic numbers.

File: app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt [hunk 29]
```diff
@@ -0,0 +1,172 @@
...
+                    leftOf(ID_BTN_PROFILE)
+                    gravity = Gravity.CENTER
+                }
+            }.lparams(dip(150), wrapContent) {
```
Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt]: WARNING: Report magic numbers. Magic number is a numeric literal that is not defined as a constant and hence it's unclear what the purpose of this number is. It's better to declare such numbers as constants and give them a proper name. By default, -1, 0, 1, and 2 are not considered to be magic numbers.

File: app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt [hunk 30]
```diff
@@ -0,0 +1,172 @@
...
+                visibility = View.VISIBLE
+            }
+            relativeLayout {
+                imageButton {
+                    id = ID_BTN_SHARE
+                    imageResource = R.drawable.ic_share
+                    backgroundResource = R.drawable.custom_bg_item_share_button
+                }.lparams(dip(50), dip(50)) {
```
Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt]: WARNING: Report magic numbers. Magic number is a numeric literal that is not defined as a constant and hence it's unclear what the purpose of this number is. It's better to declare such numbers as constants and give them a proper name. By default, -1, 0, 1, and 2 are not considered to be magic numbers.

Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt]: WARNING: Report magic numbers. Magic number is a numeric literal that is not defined as a constant and hence it's unclear what the purpose of this number is. It's better to declare such numbers as constants and give them a proper name. By default, -1, 0, 1, and 2 are not considered to be magic numbers.

File: app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt [hunk 31]
```diff
@@ -0,0 +1,172 @@
...
+                    alignParentRight()
+                    rightMargin = dip(5)
```
Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontalUI.kt]: WARNING: Report magic numbers. Magic number is a numeric literal that is not defined as a constant and hence it's unclear what the purpose of this number is. It's better to declare such numbers as constants and give them a proper name. By default, -1, 0, 1, and 2 are not considered to be magic numbers.

File: app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontal.kt [hunk 32]
```diff
@@ -0,0 +1,300 @@
...
+    private lateinit var tvCalendarTitle: TextView
+    private lateinit var rlProfile: RelativeLayout
+    private lateinit var imgBtnProfile: ImageButton
+    private lateinit var tvProfileTitle: TextView
+    private lateinit var rlShare: RelativeLayout
+    private lateinit var imgBtnShare: ImageButton
+    private lateinit var tvShareTitle: TextView
+    private lateinit var imgBtnMenu: ImageButton
+
+    private var onMenuClickListener: OnMenuClickListener? = null
```
Comment (at-vinhhuynh) [inline app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonHorizontal.kt]: please define this listener on `constructor`
and remove method `setOnMenuItemClickListener`

File: app/src/main/res/values/ids.xml [hunk 33]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <item name="floating_btn_menu_img_btn_search" type="id" />
+    <item name="floating_btn_menu_img_btn_group" type="id" />
+    <item name="floating_btn_menu_img_btn_profile" type="id" />
+    <item name="floating_btn_menu_img_btn_calendar" type="id" />
+    <item name="floating_btn_menu_img_btn_share" type="id" />
+</resources>
```
Comment (at-vinhhuynh) [inline app/src/main/res/values/ids.xml]: add a break line at end of file and add `<!-- id views on class FloatingButtonHorizontal -->`

Comment (at-toannguyen1) [reply app/src/main/res/values/ids.xml]: done!

File: app/src/main/res/values/ids.xml [hunk 34]
```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
+    <!-- id views on class FloatingButtonHorizontal -->
```
Comment (at-vinhhuynh) [inline app/src/main/res/values/ids.xml]: Have a small change, please change this comment to `<!-- View ids for FloatingButtonHorizontal -->`

File: app/src/main/res/values/ids.xml (hunk 33 above)
Comment (at-vinhhuynh) [reply app/src/main/res/values/ids.xml]: Please change this comment to `<!-- View ids for FloatingButtonHorizontal -->`

Comment (at-toannguyen1) [reply app/src/main/res/values/ids.xml]: done!

File: app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonMenu.kt [hunk 35]
```diff
@@ -0,0 +1,226 @@
...
+    private lateinit var rlShare: RelativeLayout
+    private lateinit var imgBtnShare: ImageButton
+    private lateinit var imgBtnMenu: ImageButton
+
+    init {
+        AnkoContext.createDelegate(this).apply {
+            verticalLayout {
+                lparams(wrapContent, wrapContent)
+                rlSearch = itemFloatingButton(R.id.floating_btn_menu_img_btn_search
+                        , R.drawable.custom_bg_item_search_button
```
Comment (at-vinhhuynh) [inline app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonMenu.kt]: Please break line after `,`

File: app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonMenu.kt [hunk 36]
```diff
@@ -0,0 +1,226 @@
...
+            backgroundResource = R.drawable.custom_bg_item_menu_title
+        }.lparams(wrapContent, dimen(R.dimen.height_text_view)) {
+            centerVertically()
+            rightMargin = dimen(R.dimen.margin_right_text_view)
+            leftOf(icId)
+            gravity = Gravity.CENTER
+        }
+    }
+
+    private fun onClickMenu() {
```
Comment (at-vinhhuynh) [inline app/src/main/java/vn/asiantech/way/ui/custom/FloatingButtonMenu.kt]: change method name to `onMenuClick`

**Comment by at-vinhhuynh** [comment]
@at-tienhoang 
I finish review this PR, please check it.
thank you!

**Pull request closed without merging**